src/Upgrade/Application/Strategy/Composer/Fixer/AbstractFeaturePackageFixerStep.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Application\Strategy\Composer\Fixer;
+
+use Upgrade\Application\Adapter\PackageManagerAdapterInterface;
+use Upgrade\Application\Dto\StepsResponseDto;
+use Upgrade\Application\Strategy\FixerStepInterface;
+
+abstract class AbstractFeaturePackageFixerStep implements FixerStepInterface
+{
+    /**
+     * @var string
+     */
+    protected const KEY_FEATURES = 'features';
+
+    /**
+     * @var string
+     */
+    protected const PATTERN = '/(?<features>spryker-feature\/[-\w]+).+conflicts.+/';
```

Comment:
Use more specific const name ex. FEATURE_PACKAGE_PATTERN
Use mask naming from the constant `KEY_FEATURES`